src/app/check-pop-up/check-pop-up.component.html

```diff
@@ -0,0 +1,7 @@
+<h2 mat-dialog-title>{{data?.answer}}</h2>
+
+
+<mat-dialog-actions>
```

Comment:
could you get rid of this close button and just have it display the text. If you wanted to keep an indicator of closing you could try to add an X at the top right, but I don't think it is necessary.